src/smart-components/workflow/workflow-table-helpers.js

```diff
@@ -1,6 +1,84 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Label, Flex, FlexItem } from '@patternfly/react-core';
+import { Label, Flex, FlexItem, Stack, StackItem, Button, Checkbox } from '@patternfly/react-core';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { useIntl } from 'react-intl';
+
+import WorkflowTableContext from './workflow-table-context';
+import worfklowMessages from '../../messages/workflows.messages';
+import { updateWorkflow, fetchWorkflows, moveSequence } from '../../redux/actions/workflow-actions';
+import asyncDebounce from '../../utilities/async-debounce';
+
+const debouncedFunctionsCache = {};
```

Comment:
I don't like this global variable. Does the react tabular allows to store it inside the wf table context (with setter functions ofc) without breaking it all?